src/AccessLinkGenerator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Azure.Storage.Queues;
```

Comment:
Maybe merge this file with BuzzerEventManager.cs  and create a Utilities.cs file with static helper functions